### DISCUSSION TITLE
Add 3060 ti founder edition from Notebooksbilliger

### DISCUSSION THREAD

**Discussion by fmsimp** [jef/streetmerchant#1729] [Help]
Hi, 

Im totally new to programming. I need help to add the Nvidia 3060 ti founders edition. Can someone give me a step by step introduction to add the card?

Here is the link 
https://m.notebooksbilliger.de/nvidia+geforce+rtx+3060+ti+founders+edition+695677?nbb=pn.&nbbct=1002_8_black-week#Q0S1196791132C8

Thank you!

**Comment by Jamjano** [comment]
Follow the getting started guide and then see here for settings: https://jef.codes/streetmerchant/reference/application/

So in your case you'd have to set:

`SHOW_ONLY_SERIES=3060ti`
`STORES=notebooksbilliger`
`SHOW_ONLY_BRANDS=nvidia`

**Comment by wofnull** [comment]
i submitted a patch to add the founders edition to the notebooksbilliger config ( #1744 )
It can take a bit until @jef adds this to the main repo :)

**Reply by fmsimp** [reply]
Sounds great. It would be awesome to get one for 399€ :)
